packages/developer-portal/src/components/pages/help/help.tsx

```diff
@@ -40,57 +40,65 @@ export const handleFaq = (loginIdentity: LoginIdentity) => {
+      buttonOnClick: handleGotoWelcomeGuide,
+    },
+    {
+      imgSrc: requestEndpointImg,
+      header: 'Request a Feature',
+      text: `Use this form to request a feature in either the Marketplace or Foundations API. Please note, we will
       look at all requests carefully however, we cannot guarantee all will be implemented.`,
-    buttonText: 'REQUEST',
-    buttonOnClick: handleRequestEndpoint,
-  },
-  {
-    imgSrc: reportBugImg,
-    header: 'Report a Bug',
-    text:
-      'Please report details of any bugs in relation to the Reapit Developer portal or Reapit Foundations API here. ',
-    buttonText: 'REPORT',
-    buttonOnClick: handleReportBug,
-  },
-  {
-    imgSrc: roadmapImg,
-    header: 'Roadmap',
-    text: `Want to see what we are building or check on the progress of your feature requests?
+      buttonText: 'REQUEST',
+      buttonOnClick: handleRequestEndpoint,
+    },
+    {
+      imgSrc: reportBugImg,
+      header: 'Report a Bug',
+      text:
+        'Please report details of any bugs in relation to the Reapit Developer portal or Reapit Foundations API here. ',
+      buttonText: 'REPORT',
+      buttonOnClick: handleReportBug,
+    },
+    {
+      imgSrc: roadmapImg,
+      header: 'Roadmap',
+      text: `Want to see what we are building or check on the progress of your feature requests?
       You can see our product roadmap milestones here. `,
-    buttonText: 'VIEW',
-    buttonOnClick: handleViewRoadmap,
-  },
-  {
-    imgSrc: liveChatImg,
-    header: 'Need Help?',
-    text:
-      'If you have a question that is not covered in the documentation you can ask here. Please note ' +
-      'we don’t provide chat support for Agency Cloud Developer Edition.',
-    buttonText: 'START CHAT',
-    buttonOnClick: () => handleFaq(loginIdentity),
-  },
-  {
-    imgSrc: whatNewImg,
-    header: 'What’s New',
-    text: `We are constantly working to improve your experience with the Foundations Platform. Have a look to see what
+      buttonText: 'VIEW',
+      buttonOnClick: handleViewRoadmap,
+    },
+    {
+      imgSrc: whatNewImg,
+      header: 'What’s New',
+      text: `We are constantly working to improve your experience with the Foundations Platform. Have a look to see what
       new features and fixes have been released.`,
-    buttonText: 'VIEW',
-    buttonOnClick: handleWhatsNew,
-  },
-]
+      buttonText: 'VIEW',
+      buttonOnClick: handleWhatsNew,
+    },
+  ]
+
+  if (loginIdentity.developerId && window.reapit.config.liveChatWhitelist.includes(loginIdentity.developerId)) {
+    const liveChatItem = {
+      imgSrc: liveChatImg,
+      header: 'Need Help?',
+      text:
+        'If you have a question that is not covered in the documentation you can ask here. Please note ' +
+        'we don’t provide chat support for Agency Cloud Developer Edition.',
+      buttonText: 'START CHAT',
+      buttonOnClick: () => handleFaq(loginIdentity),
+    }
+    items.splice(4, 0, liveChatItem)
```

Comment:
Are you able to do this using a different method? Using a key would be far neater and would avoid breaking the index required if someone say added a new array value that increased the index of the item you want to replace. 

So in stead of an array perhaps use an Object so that the key can be removed like so 

```ts
enum ItemKeys {
  LIVE_CHAT = 'live_chat',
}

const items: {[s: keyof  ItemKeys]: Item} = {
  [ItemKeys.LIVE_CHAT]: { ... },
  ...
}

items[ItemKeys.LIVE_CHAT] = { ... } // new item to replace with. 
//Or
delete items[ItemKeys.LIVE_CHAT]


// usage

Object.values(items).map(item => (<ItemComponent item={item} />))

```

Trying avoid a scenario where an item is added to the array altering the index of where you want to splice.